pkgs/development/tools/continuous-integration/github-runner/default.nix

```diff
@@ -294,49 +257,15 @@ stdenv.mkDerivation rec {
       --chdir $out
   '';
 
-  # Script to create deps.nix file for dotnet dependencies. Run it with
-  # $(nix-build -A github-runner.passthru.createDepsFile)/bin/create-deps-file
-  #
-  # Default output path is /tmp/${pname}-deps.nix, but can be overriden with cli argument.
-  #
-  # Inspired by passthru.fetch-deps in pkgs/build-support/build-dotnet-module/default.nix
-  passthru.createDepsFile = writeShellApplication {
-    name = "create-deps-file";
-    runtimeInputs = [ dotnetSdk nuget-to-nix ];
-    text = ''
-      # Disable telemetry data
-      export DOTNET_CLI_TELEMETRY_OPTOUT=1
-
-      rundir=$(pwd)
-
-      printf "\n* Setup workdir\n"
-      workdir="$(mktemp -d /tmp/${pname}.XXX)"
-      cp -rT "${src}" "$workdir"
-      chmod -R +w "$workdir"
-      trap 'rm -rf "$workdir"' EXIT
-
-      pushd "$workdir"
-
-      mkdir nuget_pkgs
-
-      ${lib.concatMapStrings (rid: ''
-      printf "\n* Restore ${pname} (${rid}) dotnet project\n"
-      dotnet restore src/ActionsRunner.sln --packages nuget_pkgs --no-cache --force --runtime "${rid}"
-      '') (lib.attrValues runtimeIds)}
-
-      cd "$rundir"
-      deps_file=''${1-"/tmp/${pname}-deps.nix"}
-      printf "\n* Make %s file\n" "$(basename "$deps_file")"
-      nuget-to-nix "$workdir/nuget_pkgs" > "$deps_file"
-      printf "\n* Dependency file writen to %s" "$deps_file"
-    '';
-  };
+  passthru.updateScript = ./update.sh;
```

Comment:
Does the new update script take care of all of the dependencies for the supported platforms?

Reply:
Yes. It loops over `meta.platforms` to generate an RID for every supported system, and then runs `dotnet restore` for each.

Reply:
Great, thanks!